### ISSUE TITLE
Better highlighting of turn directions

### ISSUE THREAD

**Issue by Max1234-Ita** [osmandapp/OsmAnd#16209]
### Description
I'm trying to help a friend who's a bus driver.
He's often facing situations like the one depicted in Picture_1: driving along a motorway/trunk, he needs to take an exit that can only be reached following a separated highway parallel to the main road: The distance to be covered on it is often 500 m or more. 

The first turn, to leave the motorway is not very visible on the map since the roads are very close each other and the driver might miss it; When the turn to the exit is finally announced it's too late, since the 2 roads are segregated by a guard rail and there's no way to get through (see Picture_2 and Picture_3).

A good solution would be having the map temporary zoomed in at the turn indication, and/or the turn direction panel flashing for a few seconds: so far, the only highlight is provided by the indicator arrow becoming green (normally it's yellow), not very noticeable if you're concentrated on the road and driving at 70+ Km/h .

The last attached file is a .zip containing a .gpx track over the same area shown in the pictures, hope it can be useful.
Thanks for your attention!
Max

![Picture_1](https://user-images.githubusercontent.com/15315381/211791843-95cab3b0-da73-42d6-9cbd-b2bd52b62121.jpg)

![Picture_2](https://user-images.githubusercontent.com/15315381/211793405-3026149a-8d77-40ce-a319-8924714e242e.jpg)

![Picture_3](https://user-images.githubusercontent.com/15315381/211797363-90e001ca-3d99-4b86-84a3-c1128bd4afd9.jpg)

![image](https://user-images.githubusercontent.com/15315381/211809578-571fa405-e9e6-4da5-ab2b-e40fb7a6cd05.png)
(sorry for the Google image :) )

<br/>
<br/>

[Wed 11 Jan 2023.zip](https://github.com/osmandapp/OsmAnd/files/10392276/Wed.11.Jan.2023.zip)

<br/>

### Describe the solution you'd like
- Auto-zoom the map on the target area when a turn direction is emitted;
- Allow to customise more the time advance for the voice announcements;
- Make the turn direction panel in the display more noticeable, i.e. flashing or inverting the colors for a short time, when an announcement is emitted (many drivers don't use voice prompts  to lower distractions but they need to know when the turn is approaching);

### Describe alternatives you've considered
- Tried setting the Announcement time to _Early_ in Configure profile -> Navigation settings -> Voice prompts, not helping very much

- Also the auto-zoom on slow speeds feature is not helping, since in the described situation the road change doesn't occur at slow speeds (if the driver is unaware of the lane/road change, he's likely not slowing down).

**Comment by scaidermern** [comment]
I'm wondering why you are using the browse map profile and not the car profile?

With the car profile the turn instructions are pretty clear in my opinion:
![photo_2023-01-11_14-35-10](https://user-images.githubusercontent.com/1040784/211819759-2543cd4c-d94d-469a-83e6-00697384e2b9.jpg)

**Comment by Max1234-Ita** [comment]
> I'm wondering why you are using the browse map profile and not the car profile?
> 
> With the car profile the turn instructions are pretty clear in my opinion: ![photo_2023-01-11_14-35-10](https://user-images.githubusercontent.com/1040784/211819759-2543cd4c-d94d-469a-83e6-00697384e2b9.jpg)

You're right, I used that profile to prepare the gpx I was not navigating at that moment.. 
Anyway, i agree, the instructions are pretty clear... if you look at the screen
But if you are concentrated onto the road (like a driver is supposed to be) the green arrows are not so noticeable, and you may miss them.
